_posts/dart-cuts.md

```diff
@@ -45,52 +44,52 @@ and finally
 
 ## Rowlett
-July 2nd - Rowlett's city council put forward a resolution with the exact same wording as Plano's. Plano's presentor seemed more like a politician. They talk about going to the Texas legislature, include facts that support them, exclude things that don't, and use careful wording to makes you think something without them actually saying it. Rowlett on the other hand is a bit more rough. The presentation included numerous errors, perplexing assumptions, and then projections based off of those previous errors that give an even more egregious result. But without the relevant background knowledge the council voted unanimously in support of it anyways.
+July 2nd - Rowlett's city council put forward a resolution with identical wording to Plano's. Plano's presenter sounded like a politician, talking about going to the Texas legislature, including only facts that support their narrative, and using careful wording to imply certain things without actually saying them. On the other hand, Rowlett's presentation was a bit rougher. It included numerous errors, perplexing assumptions, and projections based off of those previous errors that gave an even more egregious result. But without the relevant background knowledge, the council voted unanimously in support of it anyways.
 
-A slide by slide fact-check of Rowlett's presentation can be seen [here](https://docs.google.com/presentation/d/1GKMZaz76c3Sck3ZZscDksqYOR-HvFt7lSw7mv2DeDCg)
+A slide-by-slide fact-check of Rowlett's presentation can be seen [here](https://docs.google.com/presentation/d/1GKMZaz76c3Sck3ZZscDksqYOR-HvFt7lSw7mv2DeDCg).
 
 ## Irving
-July 11th - Irving city council had DART budget cuts on the agenda. At first it was under the consent agenda, a part of the agenda that is supposed to contain rote items that can be voted on for approval in bulk. Many citizens reached out and asked for them to remove it from the consent agenda so it can have the full discussion it deserves. Irving obliged and many citizens attended the meeting urging them to vote against it. Similar to Plano, Irving council members made vague arguments of fiscal responsibility and examples of DART funding instead paying for police and fire, or lower property taxes. The vote passed unanimously.
+July 11th - Irving city council had DART budget cuts on the agenda. At first, it was under the consent agenda, a part of the agenda that is supposed to contain rote items that can be voted on for approval in bulk. Many residents reached out asking for them to remove it from the consent agenda so it could've had the full discussion it deserved. Irving obliged, and citizens flocked to the meeting urging the council to vote against the resolution. Similar to Plano's council meeting, however, Irving council members made vague arguments of fiscal responsibility and examples of the cut DART funding instead paying for police and fire, or lower property taxes. The vote passed unanimously.
```

Comment:
I don’t think “could’ve had” is the right tense here, maybe “could have”

Reply:
I rewrote the phrasing connection to avoid the weird tense.